python/tvm/relay/op/tensor.py

```diff
@@ -345,3 +330,41 @@ def ones_like(data):
         The computed result.
     """
     return _make.ones_like(data)
+
+def concatenate(data, axis=0):
```

Comment:
Let us consider remove default axis in concatenate for now, The main reason is that historically concat usually defaults to axis=1, this new default will likely cause some confusion.